app/experimenter/targeting/constants.py

```diff
@@ -628,6 +628,16 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+BACKGROUND_TASK_NOTIFICATION = NimbusTargetingConfig(
+    name="Background task notification",
+    slug="Background_task_notification",
+    description="Firefox running a background task",
+    targeting="isBackgroundTaskMode",
+    desktop_telemetry="",
+    sticky_required=True,
```

Comment:
Question: Do you need` sticky required` to be `True`?

Reply:
No.  In fact, it would be fascinating if we witnessed unenrollment from these experiments due to `targeting-mismatch`, i.e., `isBackgroundTaskMode` oscillating.

Reply:
I copied the stickiness of `WINDOWS_10_PLUS_BACKGROUND_TASK_NOTIFICATION_` but doesn't need to be.